blazor/diagram/swimlane/phase.md

```diff
@@ -0,0 +1,242 @@
+* The appearance of Phase header can be customized by using the `Style` properties.
+
+The following code example explains how to define a Phase header and its customization.
+
+```cshtml
+@using Syncfusion.Blazor.Diagram
+
+<SfDiagramComponent Height="600px" Swimlanes="@SwimlaneCollections" />
+
+@code
+{
+    //Define diagram's swimlane collection
+    DiagramObjectCollection<Swimlane> SwimlaneCollections = new DiagramObjectCollection<Swimlane>();
+
+    protected override void OnInitialized()
+    {
+        // A swimlane is created and stored in the swimlanes collection.
+        Swimlane swimlane = new Swimlane()
+            {
+                Header = new SwimlaneHeader()
+                {
+                    Annotation = new ShapeAnnotation()
+                    {
+                        Content = "SALES PROCESS FLOW CHART"
+                    },
+                    Height = 50,
+                },
+                OffsetX = 400,
+                OffsetY = 200,
+                Height = 150,
+                Width = 450,
+                Phases = new DiagramObjectCollection<Phase>()
+                {
+                    new Phase()
+                    {
+                        Width = 450,
+                        Header = new SwimlaneHeader()
+                        {
+                            Annotation = new ShapeAnnotation(){Content = "Phase 1", Style = new TextStyle(){ Color = "White", TextDecoration = TextDecoration.Underline, Italic = true, Bold = true} },
+                            Height = 30
+                        },
+                        Style = new TextStyle(){Fill = "Teal"},
+                    }
+                }
+            };
+        // Add swimlane
+        SwimlaneCollections.Add(swimlane);
+    }
+}
+```
+![Phase Header](Swimlane-images/Swimlane_Phase_Header.PNG).
+
+You can download a complete working sample from [GitHub]()
+
+## Header Selection and Resize
+
+ * You can select the individual phase header by clicking on the header twice. On the first click, you can select the respective phase. 
+
+ * You can able to resize the individual phase header. While resizing the phase, it will maintain 20px distances from Lane children.
+
+  * When a element is resized, the [SizeChanging](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Diagram.SfDiagramComponent.html#Syncfusion_Blazor_Diagram_SfDiagramComponent_SizeChanging) and [SizeChanged](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Diagram.SfDiagramComponent.html#Syncfusion_Blazor_Diagram_SfDiagramComponent_SizeChanged) events get triggered.
+
+ The following image shows how to select and resize the phase header.
+
+![Header Select and Resize](Swimlane-images/Header_Selection_Resize.gif).
+
+## Phase header editing
+
+The diagram provides support for editing phase headers at runtime. You can achieve header editing by using the double click event. Double-clicking the header label enables the editing of that specific header. 
```

Comment:
The diagram provides support for editing phase headers at runtime. You can achieve header editing by using the double-click event. Double-clicking the header label enables the editing of that specific header.